src/controller/python/chip/ble/darwin/Scanning.mm

```diff
@@ -23,10 +24,12 @@ @interface ChipDeviceBleScanner : NSObject <CBCentralManagerDelegate>
 @property (assign, nonatomic) PyObject * context;
 @property (assign, nonatomic) DeviceScannedCallback scanCallback;
 @property (assign, nonatomic) ScanCompleteCallback completeCallback;
+@property (assign, nonatomic) ScanErrorCallback errorCallback;
```

Comment:
The value of this property is not used anywhere.  Should it be?

Reply:
You are right, I should call it in stopTimeoutReached